### DISCUSSION TITLE
无法找到依赖该怎么决绝啊佬们

### DISCUSSION THREAD

**Discussion by ghost** [Openwrt-Passwall/openwrt-passwall2#94] [Q&A] [answered]
WARNING: Makefile 'package/feeds/passwall2/luci-app-passwall2/Makefile' has a dependency on 'tcping', which does not exist
WARNING: Makefile 'package/feeds/passwall2/luci-app-passwall2/Makefile' has a dependency on 'xray-core', which does not exist
WARNING: Makefile 'package/feeds/passwall2/luci-app-passwall2/Makefile' has a dependency on 'v2ray-geoip', which does not exist
WARNING: Makefile 'package/feeds/passwall2/luci-app-passwall2/Makefile' has a dependency on 'v2ray-geosite', which does not exist
WARNING: Makefile 'package/feeds/passwall2/luci-app-passwall2/Makefile' has a dependency on 'brook', which does not exist
WARNING: Makefile 'package/feeds/passwall2/luci-app-passwall2/Makefile' has a dependency on 'hysteria', which does not exist
WARNING: Makefile 'package/feeds/passwall2/luci-app-passwall2/Makefile' has a dependency on 'naiveproxy', which does not exist
WARNING: Makefile 'package/feeds/passwall2/luci-app-passwall2/Makefile' has a dependency on 'shadowsocks-rust-sslocal', which does not exist
WARNING: Makefile 'package/feeds/passwall2/luci-app-passwall2/Makefile' has a dependency on 'shadowsocksr-libev-ssr-local', which does not exist
WARNING: Makefile 'package/feeds/passwall2/luci-app-passwall2/Makefile' has a dependency on 'shadowsocksr-libev-ssr-redir', which does not exist
WARNING: Makefile 'package/feeds/passwall2/luci-app-passwall2/Makefile' has a dependency on 'shadowsocksr-libev-ssr-server', which does not exist
WARNING: Makefile 'package/feeds/passwall2/luci-app-passwall2/Makefile' has a dependency on 'simple-obfs', which does not exist
WARNING: Makefile 'package/feeds/passwall2/luci-app-passwall2/Makefile' has a dependency on 'v2ray-core', which does not exist
WARNING: Makefile 'package/feeds/passwall2/luci-app-passwall2/Makefile' has a dependency on 'v2ray-plugin', which does not exist

**Comment by motiyb** [comment, marked as answer]
我也碰到这个问题，找到答案了来回一下，需要把package加到feeds里。

具体操作是把下面的命令加到 feeds.conf.default 里。 
src-git PWpackages https://github.com/xiaorouji/openwrt-passwall.git;packages


我是这样操作的就可以了，有不对的地方请大佬们补充。

更新: PW1的discussion里面有编译的文档，貌似我之前在网上查到的解决方法就是从这里面来的。
[https://github.com/xiaorouji/openwrt-passwall/discussions/1603](url)
